cmd/package-operator-manager/bootstrap/bootstrap_test.go

```diff
@@ -90,6 +90,7 @@ func TestBootstrapperBootstrap(t *testing.T) {
 	assert.Equal(t, "noxxx", os.Getenv("NO_PROXY"))
 }
 
+//nolint:fatcontext
```

Comment:
I'd suggest moving this directly to the offending line and/or explaining why the linter is excluded for the function.

Reply:
Applied